response_operations_ui/forms.py

```diff
@@ -40,6 +40,22 @@ class SearchForm(FlaskForm):
     submit = SubmitField('Search')
 
 
+class EditContactDetailsForm(FlaskForm):
+    first_name = StringField('first_name')
```

Comment:
Should we have some kind of validation in this form?

Reply:
This will be part of tech debt @davidmort made this decision, validation is currently done in the html